index.bs

```diff
@@ -1048,6 +1048,7 @@ Possible values are:
 <ul dfn-for="source debug data type">
 <li>"<dfn><code>source-destination-limit</code></dfn>"
 <li>"<dfn><code>source-destination-rate-limit</code></dfn>"
+<li>"<dfn><code>source-destination-wide-rate-limit</code></dfn>"
```

Comment:
IMO `wide` is a bit ambiguous here. Can we think of another way of expressing this?